apps/cli/README.md

```diff
@@ -1,10 +1,68 @@
-## ledger-live CLI tools
+# ledger-live CLI
 
 > Please be advised this software is experimental and shall not create any obligation for Ledger to continue to develop, offer, support or repair any of its features. The software is provided “as is.” Ledger shall not be liable for any damages whatsoever including loss of profits or data, business interruption arising from using the software.
 
+This package wraps functionality from `@ledgerhq/live-common` into a suite of tools that can be used directly from the terminal
+
+# Usage
+
+## Install the CLI
+
+```bash
+npm i --global @ledgerhq/live-cli
```

Comment:
(there might be some doc to adapt following this)

Reply:
@gre I think we changed the name because it conflicted with the root package.json being called ledger-live as well. This was an arbitrary choice and I do not mind changing the name. However we started publishing to npm with this name already. If it's fine with everyone, we can keep it that way and update doc